presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -96,7 +96,14 @@ private static void verifyJavaVersion()
         }
 
         JavaVersion version = JavaVersion.parse(javaVersion);
-        if (version.getMajor() == 8 && version.getUpdate().isPresent() && version.getUpdate().getAsInt() >= 161) {
+
+        if (version.getMajor() >= 11) {
+            return;
+        }
+
+        if ((version.getMajor() == 8 && version.getUpdate().isPresent() && version.getUpdate().getAsInt() >= 161) ||
+                (version.getMajor() > 8 && version.getMajor() < 11)) {
+            warnRequirement("Future versions of Presto will require Java 11+ (found: %s)", javaVersion);
```

Comment:
`if (version.getMajor() >= 9) {` below is redundant

Reply:
Well... we dont support 9 or 10 I assume.. since they are not LTS

Reply:
Yes, good catch. Fixed.